cmd/crictl/container.go

```diff
@@ -351,7 +351,12 @@ var removeContainerCommand = &cli.Command{
 		}
 
 		if len(ids) == 0 {
-			return cli.ShowSubcommandHelp(ctx)
+			if ctx.Bool("all") {
+				logrus.Info("No containers to remove")
+				return nil
```

Comment:
nit: since we return here, we don't need the `else` below.

Reply:
you want me to change all `if A return B else return C` ? not used to the style you recommend but I don't really have preferences

Reply:
yeah please do, its a style preference of mine, thanks :)

Reply:
done